graphql-test-utils/graphql-models/custom-application-installation.js

```diff
@@ -0,0 +1,11 @@
+import { Factory } from 'rosie';
```

Comment:
FYI: as a follow up we should rewrite these test-data models to use our new setup.